samples/snippets/src/main/resources/us-states.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type":"record",
+  "name":"State",
+  "namespace":"pubsub",
```

Comment:
I should change "namespace" to "utilities" perhaps and have the generated class sit in `samples/snippets/src/main/java/utilities/State.java`.